spec/spec.yml

```diff
@@ -1430,6 +1430,119 @@ paths:
           description: 'Unexpected error'
           schema:
             $ref: '#/definitions/Error'
+  /projects/{uuid}/annotation-groups/:
+    x-resource: Projects
+    get:
+      summary: 'Get annotation groups belonging to a project'
+      tags:
+        - Imagery
+      parameters:
+        - $ref: '#/parameters/uuid'
+      responses:
+        200:
+          description: 'Array containing annotation groups for this project'
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/AnnotationGroup'
+        403:
+          description: 'Insufficient permissions or object does not exist'
+          schema:
+            $ref: '#/definitions/Error'
+        default:
+          description: 'Unexpected error'
+          schema:
+            $ref: '#/definitions/Error'
+    post:
+      summary: 'Create annotation group for this project'
+      tags:
+        - Imagery
+      parameters:
+        - $ref: '#/parameters/uuid'
+        - name: annotationGroup
+          in: body
+          required: true
+          schema:
+            $ref: '#/definitions/AnnotationGroupCreate'
+      responses:
+        201:
```

Comment:
This _should_ be correct, but is not -- right now a `200` comes back. I'll fix that elsewhere.